R/mice.impute.pmm.R

```diff
@@ -104,13 +105,36 @@
 #' )
 #' abline(0, 1)
 #' cor(y, yimp, use = "pair")
+#' 
+#' # Use blots to exclude different values per column
+#' # Create blots object
+#' blots <- make.blots(boys)
+#' # Exclude ml 1 through 5 from tv donor pool
+#' blots$tv$exclude <- c(1:5)
+#' # Exclude 100 random observed heights from tv donor pool
+#' blots$hgt$exclude <- sample(unique(boys$hgt), 100) 
+#' imp <- mice(boys, method = "pmm", print = FALSE, blots = blots, seed=123)
+#' blots$hgt$exclude %in% unlist(c(imp$imp$hgt)) # MUST be all FALSE 
+#' blots$tv$exclude %in% unlist(c(imp$imp$tv)) # MUST be all FALSE 
 #' @export
-mice.impute.pmm <- function(y, ry, x, wy = NULL, donors = 5L,
+mice.impute.pmm <- function(y, ry, x, exclude = -99999999, wy = NULL, donors = 5L,
                             matchtype = 1L, ridge = 1e-05,
```

Comment:
Please place the `exclude` argument after `matchtype` so that generic `wy` and the long-time methodology `donors` and `matchtype` arguments remain in place. Also update in roxygen helps.

Reply:
Done.